cmake/oss.cmake.in

```diff
@@ -42,10 +42,38 @@ if (@CMAKE_CXX_COMPILER_ID@ MATCHES "GNU")
   set(seastar_cxx_flags "-Wno-error$<SEMICOLON>-fcoroutines")
 endif()
 
+if ("@CMAKE_CXX_COMPILER_ID@" MATCHES "Clang")
```

Comment:
Note (no action required, just for posterity): This is different to the other build, which uses `CMAKE_CXX_COMPILER` which always fails the check (but creates build problems if "fixed") - this one will actually enable `llvm-ar` for clang builds, but that's not a tested scenario, I'd be surprised if it works.

Reply:
I am going to add issue for that after merge